CHANGELOG.md

```diff
@@ -1,7 +1,24 @@
 x.y.z Release notes (yyyy-MM-dd)
 =============================================================
 ### Enhancements
-* None.
+* The following classes & aliases have been renamed:
+- RLMSyncUser -> RLMUser
+- RLMAppCredential -> RLMCredential
+- RealmApp -> App
+- RLMUserAPIKeyProviderClient -> RLMAPIKeyAuth
+- RLMUsernamePasswordProviderClient -> RLMEmailPasswordAuth
+
+* The following functionality has also moved to the User
+
+- callFunctionNamed
```

Comment:
Should use proper objc/swift syntax for these.

`-[RLMApp callFunctionNamed:]` -> `-[RLMUser callFunctionNamed:]`
`RealmApp.functions` -> `User.functions`

Reply:
You should do this for all of the above and below. It would also be helpful to add a brief statement on "why" we're doing this.